src/main/scala/uk/gov/nationalarchives/referencegenerator/Lambda.scala

```diff
@@ -57,7 +56,7 @@ class Lambda extends RequestHandler[APIGatewayProxyRequestEvent, APIGatewayProxy
 }
 
 object Lambda {
-
+  val logger: Logger = Logger[Lambda]
```

Comment:
Assuming the `logger` class wasn't getting constructed when the `Lambda` class was being created on invocation of the lambda?